handlers/main.yml

```diff
@@ -15,6 +15,12 @@
     state: restarted
     enabled: yes
 
+# https://github.com/zalando/patroni/blob/master/docs/watchdog.rst
+- file:

```

Comment:
This task is excessive because correct device permissions are achieved by [patroni-watchdog.service](https://github.com/kostiantyn-nemchenko/ansible-role-patroni/blob/master/templates/patroni-watchdog.service.j2#L11) at system start up.